gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -340,9 +353,7 @@ local function CreateEquipmentList(t)
 				-- Favorites marker icon
 				ic.favorite = false
 
-				local favorites = GetFavorites(steamid, currole)
-
-				if favorites and IsFavorite(favorites, item.id) then
+				if equipFavModel:Find({steamid, currole, item.id}) then
```

Comment:
check if an object of the model gets created if it does it exists in the database